grafana/resource_cloud_stack.go

```diff
@@ -349,7 +349,9 @@ func waitForStackReadiness(ctx context.Context, d *schema.ResourceData) diag.Dia
 		}
 		defer resp.Body.Close()
 		if resp.StatusCode != 200 {
-			return resource.RetryableError(errors.New("stack is not ready yet"))
+			buf := new(bytes.Buffer)
+			buf.ReadFrom(resp.Body)
```

Comment:
This call can [potentially panic](https://pkg.go.dev/bytes#Buffer.ReadFrom), which would be less than ideal but it will be managed by `go test` (sort of) so there's that. I wonder it it would make sense to check the error for other than `io.EOF` 🤔

Reply:
Oops, I checked the error